### PR TITLE
Update Foxy Github actions

### PR DESCRIPTION
**Pull request by ahcorde**
I had some troubles with the Github Actions on Foxy in particular this PR https://github.com/ros2/rcpputils/pull/103

This should fix the problem

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -17,10 +17,11 @@ jobs:
       matrix:
           os: [ubuntu-18.04]
     steps:
-    - uses: ros-tooling/setup-ros@0.0.13
-    - uses: ros-tooling/action-ros-ci@0.0.13
+    - uses: ros-tooling/setup-ros@0.0.25
+    - uses: ros-tooling/action-ros-ci@0.0.18
```
Comment (clalancette) [inline .github/workflows/test.yml:21]: I'll suggest that you stick to the same versions as master here for consistency (`setup-ros@0.0.23` and `action-ros-ci@0.0.17`).

If we want to upgrade further, we should probably do it on master first and then backport to Foxy.

**Review by clalancette — CHANGES_REQUESTED** [review]
One thing to fix.

On a separate note, I'm wondering if we should just remove the Github workflow altogether.  We already have PR jobs, and we require CI on all merged PRs anyway, so it seems redundant.  Thoughts?

File: .github/workflows/test.yml (hunk 1 above)
Comment (ahcorde) [reply .github/workflows/test.yml]: Updated: https://github.com/ros2/rcpputils/pull/104/commits/1860573f9d93c60cf3e484a5972709337fb442e9

We should probably remove the Github workflow in master (first), right ? and backport the change.

**Comment by dirk-thomas** [comment]
Why does this repo not use the PR CI offered by the ROS buildfarm?

**Comment by ahcorde** [comment]
The ownership of the repo was recently transfer to the Open Robotics https://github.com/ros2/rcpputils/pull/100#event-3830528900. It was managed by AWS, they configure Github Actions for theirs repos.

Anyhow I'm planning to release a new version I can include the CI offered by the ROS buildfarm in Rolling and Foxy

File: .github/workflows/test.yml (hunk 1 above)
Comment (clalancette) [reply .github/workflows/test.yml]: > We should probably remove the Github workflow in master (first), right ? and backport the change.

Yeah, that would be my suggestion.

**Review by clalancette — APPROVED** [review]

**Pull request merged**